support/utils/command.go

```diff
@@ -82,3 +83,14 @@ func RunCommand(command string) *exec.Proc {
 func FetchCommandOutput(command string) string {
 	return commandRunner.Run(command)
 }
+
+func FetchCommandStdoutStderr(command string) (stdout, stderr string, err error) {
```

Comment:
Hi @heylongdacoder couple of suggestions:

1. The "Fetch" prefix does not really describe that the command is "executed" (I see there is precedance, so maybe Ok)
2. Name suggestion: "RunWithSeparatedOutput" ?

Also, whichever name you select, please add a comment over the function to describe its operation.

Reply:
@vladimirvivien agreed. Since we have "RunCommand", I renamed it to "RunCommandWithSeperatedOutput", is this better or function name too long? 😆 

I also added some comment on the functions. 😄